qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -1,23 +1,32 @@
 - single_driver_install:
     type = single_driver_install
     only Windows
-    cdroms += " virtio winutils"
-    kill_vm = yes
-    login_timeout = 360
-    need_uninstall = no
-    driver_install_timeout = 600
-    tmp_folder = C:\tmp
-    guest_alias = "Win7-32:w7\x86,Win7-64:w7\amd64,Win8-32:w8\x86,Win8-64:w8\amd64,Win8-32.1:w8.1\x86,Win8-64.1:w8.1\amd64,Win10-32:w10\x86,Win10-64:w10\amd64,Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8R2\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12R2\amd64,Win2016-64:2k16\amd64"
-    driver_install_cmd = "python WIN_UTILS:\win_driver_install.py %s %s %s %s %s"
-    show_file_ext_cmd = "reg add HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced /v HideFileExt /t REG_DWORD /d 0 /f"
+    # cert_files: a list of certification files whose element is a pair of
+    #             cert store and cert file path.
+    #             (e.g. 'trustedpublisher=X:\virtio-win.cer')
+    cert_files = ""
+    variants:
+        - virtio_win_iso_media:
+            virtio_win_media_type = iso
+            cdroms += " virtio"
+        - virtio_win_vfd_media:
+            only with_netkvm with_viostor with_vioscsi
+            virtio_win_media_type = vfd
+            floppies = "virtio"
+            i386:
+                only Win7 Win8 Win2008
```

Comment:
Could we delete line17 and line20? As this case is already only windows and win2012/2016 has no i386 platform.

Reply:
Since VFD images only contain drivers of the listed types of Windows system, we would get errors if here do not provide the filter.

Reply:
I removed line20 and run job on win2016, the job passed as well.
Could you have a double check?

Reply:
OK, after our discussion, the issue is upstream virtio-win do not contain win10's vfd image.

Since our downstream version has it, I will remove this only filter.